index.html

```diff
@@ -30,9 +33,13 @@ <h2 class="support-text-2 default-custom-fonts lets-text-font">LETS CONNECT</h2>
         </div>
     </section>
     <!-- Adding the contact cards -->
+    <!-- <div class="main-cont-grid"> -->
     <section class="grid-container">
-      <div class="flex-item">
-        <img class="image" src="images/Portfolio.svg" alt="Tonic">
+      <div class="grid-item grid-item1 flex-item g1">
+        <img class="image tonicA" src="images/Portfolio.svg" alt="Tonic">
+        <img class="image desktop" src="images/desaktop-2.svg" alt="desktop">
+      </div>
+      <div class="grid-item grid-item2 flex-item g1">
         <p class="tonic">Tonic</p>
         <ul class="list-item default-list-style">
```

Comment:
[OPTIONAL]
- [ ] For accessibility purposes, kindly make sure all your card titles as grabbed in a header tag `hx` and not a paragraph as is the case in your code